src/NumericalAlgorithms/Spectral/SwshCollocation.hpp

```diff
@@ -8,11 +8,20 @@
 #include <sharp_cxx.h>
 
 #include "NumericalAlgorithms/Spectral/ComplexDataView.hpp"
+#include "Utilities/ForceInline.hpp"
 #include "Utilities/Gsl.hpp"
 
 namespace Spectral {
 namespace Swsh {
 
+/// \ingroup SwshGroup
+/// \brief Convenience function for determining the number of spin-weighted
+/// spherical harmonic collocation values that are stored for a given `l_max`.
```

Comment:
Perhaps there should be a `libsharp` somewhere in this comment because the number of collocation points is determined by libsharp?